pom.xml

```diff
@@ -296,6 +324,11 @@
                                     <include>org.checkerframework:checker-qual</include>
                                     <include>org.checkerframework:checker-compat-qual</include>
                                     <include>org.codehaus.mojo:animal-sniffer-annotations</include>
+                                    <include>com.fasterxml.jackson.core:*</include>
+                                    <include>com.fasterxml.jackson.dataformat:*</include>
+                                    <include>com.fasterxml.jackson.module:*</include>
+                                    <!-- Include paranamer required by jackson-module-scala -->
```

Comment:
Looks like `com.fasterxml.jackson.dataformat:jackson-dataformat-yaml` has a dependency on `org.yaml:snakeyaml`. That should be included here as well.